creator-node/src/routes/tracks.js

```diff
@@ -27,7 +27,7 @@ const {
 } = require('../middlewares')
 
 const { getCID } = require('./files')
-const { decode } = require('../hashids.js')
+const { decode } = require('../hashids')
```

Comment:
we have to omit the extension because locally it's reading the ts file but in the prod image it's using a transpiled version of js. the transpiled version should be 1-1 to the ts so functionally the same